src/server/scripts/Northrend/Gundrak/boss_drakkari_colossus.cpp

```diff
@@ -112,6 +113,20 @@ struct boss_drakkari_colossus : public BossAI
         Initialize();
     }
 
+    void EnterEvadeMode(EvadeReason why) override
+    {
+        if (why == EVADE_REASON_NO_HOSTILES)
+        {
+            if (isFrozen)
+                return;
+
+            DoZoneInCombat();
```

Comment:
What is this supposed to do? Why are you forcing combat with zone on evade (!) when not frozen during elemental phase?

Reply:
Very rarely while testing, I had a case that I couldn't really reproduce consistently where the boss would reset upon unfreezing instead of re-engaging player(s), that's why I moved it there

Reply:
so, did we get stucked here ?